### DISCUSSION TITLE
How to UTF-8 decode in Enhanced table like the documents table

### DISCUSSION THREAD

**Discussion by Thehunter74** [fbaligand/kibana-enhanced-table#322] [Q&A]
Hello, Wanted to see if there was a way to get the Enhanced table to decode my subject lines that are sometimes encoded in utf-8. The documents table seems to do this automatically but the enhanced table doesn't seem to be doing the same thing. Is there a way to enable this so my subject lines will display decoded instead of encoded by utf-8?

**Comment by fbaligand** [comment]
Well, I'm quite surprised by your question.
Text is stored in Elasticsearch in UTF-8 encoding. And there is no decoding to do in the visualization.

Personally, I have to deal with with correct encoding reading at source parsing (Filebeat, Logstash) when I read a file. But not after because all is processed in UTF-8 (both Elasticsearch & Kibana).

So I don't understand how you can have such a problem.
